DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -644,92 +892,129 @@ namespace pat {
     bool isPhoton() const override { return false; }
     bool isConvertedPhoton() const override { return false; }
     bool isJet() const override { return false; }
-    bool isGoodEgamma() const { return (qualityFlags_ & egammaFlagsMask) !=0; }
+    bool isGoodEgamma() const { return (qualityFlags_ & egammaFlagsMask) != 0; }
 
     // puppiweights
     void setPuppiWeight(float p, float p_nolep = 0.0);  /// Set both weights at once (with option for only full PUPPI)
     float puppiWeight() const;                          /// Weight from full PUPPI
     float puppiWeightNoLep() const;                     /// Weight from PUPPI removing leptons
-    
+
     // for the neutral fractions
-    void setRawCaloFraction(float p);                      /// Set the raw ECAL+HCAL energy over candidate energy for isolated charged hadrons
-    float rawCaloFraction() const { return (rawCaloFraction_/100.); }    /// Raw ECAL+HCAL energy over candidate energy for isolated charged hadrons
-    void setRawHcalFraction(float p);                      /// Set the fraction of Hcal needed isolated charged hadrons
-    float rawHcalFraction() const { return (rawHcalFraction_/100.); }    /// Fraction of Hcal for isolated charged hadrons
-    void setCaloFraction(float p);                      /// Set the fraction of ECAL+HCAL energy over candidate energy
-    float caloFraction() const { return (caloFraction_/100.); }    /// Fraction of ECAL+HCAL energy over candidate energy
-    void setHcalFraction(float p);                      /// Set the fraction of Hcal needed for HF, neutral hadrons, and charged particles
-    float hcalFraction() const { return (hcalFraction_/100.); }    /// Fraction of Hcal for HF, neutral hadrons, and charged particles
-    
-
-     // isolated charged hadrons
-    void setIsIsolatedChargedHadron(bool p);                      /// Set isolation (as in particle flow, i.e. at calorimeter surface rather than at PV) flat for charged hadrons
-    bool isIsolatedChargedHadron() const { return isIsolatedChargedHadron_; }    /// Flag isolation (as in particle flow, i.e. at calorimeter surface rather than at PV) flag for charged hadrons
+    void setRawCaloFraction(
+        float p);  /// Set the raw ECAL+HCAL energy over candidate energy for isolated charged hadrons
+    float rawCaloFraction() const {
+      return (rawCaloFraction_ / 100.);
+    }                                  /// Raw ECAL+HCAL energy over candidate energy for isolated charged hadrons
```

Comment:
@fabiocos @davidlange6 
this is a case where the post-clang-format result looks worse than the original.
The comment should be closer or on the same line as the method definition, not at the end of the scope